.gitignore

```diff
@@ -168,3 +168,7 @@ symengine/symengine_config_cling.h
 # IDE related
 /.idea
 .*cache/
+.vscode/
+
+#Personal
+my/
```

Comment:
I wouldn't add anything personal to the .gitignore. (IDE-related stuff should be fine though.)

There are other ways of accomplishing the same thing. See for example: https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file

Reply:
Ok. Sympy's .gitignore has a /my, so I thought it would be fine to add it here as well. I'll remove it and look into what you've linked.